examples/minimal_cxx_qt_qml/src/tests/myobject/tst_myobject.qml

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+// SPDX-FileContributor: Gerhard de Clercq <gerhard.declercq@kdab.com>
```

Comment:
Please add yourself to SPDX-FileContributor here any elsewhere where it makes sense :-)

Reply:
Done